test/integration/db_integration_test.go

```diff
@@ -43,17 +43,21 @@ func (a *dbIntegrationSuite) SetupTest() {
 func (a *dbIntegrationSuite) TearDownTest() {
 	err := a.be.Stop()
 	require.NoError(a.T(), err)
+	time.Sleep(time.Second * 1)
```

Comment:
why do we need to sleep here?

Reply:
extractor's channels continue to listen for some reason after stopping all BE layers. Without that sleep the error may occur (see https://github.com/insolar/block-explorer/runs/738128508?check_suite_focus=true):
```goroutine 1000 [running]:
github.com/insolar/block-explorer/etl/extractor.(*PlatformExtractor).getJetDrops.func1(0x1aafd00, 0xc000120010, 0xc00001e340, 0xc000360de0, 0x1a89840, 0xc000124bb0, 0xc000360e44, 0xc000360e40, 0xffffffff0046f900)
	/home/runner/work/block-explorer/block-explorer/go/src/github.com/insolar/block-explorer/etl/extractor/platform_impl.go:107 +0x2ea
created by github.com/insolar/block-explorer/etl/extractor.(*PlatformExtractor).getJetDrops
	/home/runner/work/block-explorer/block-explorer/go/src/github.com/insolar/block-explorer/etl/extractor/platform_impl.go:76 +0x14a
FAIL	github.com/insolar/block-explorer/test/integration	35.385s```

Reply:
new bug created, link attached